### PR TITLE
Initialize bookstore PR

### PR DESCRIPTION
**Pull request by adamteddychang**
In this branch I : 
- Removed most files from boilerplate
- Added React Redux
- Structured files with ducks pattern
- Added several components
- Used React Routers
- [Live Link](https://adamteddychang-bookstore.netlify.app/)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        <Navbar />
+        <Routes>
+          <Route
+            exact
+            path="/"
+            element={(
+              <div>
+                <BooksList />
+                <InputBook />
+              </div>
```
Comment (SajjadAhmad14) [inline src/App.js]: Please add the `BooksList` component as the default page and add a separate route for the `InputBook` component which should render a form on click.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Adam, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
Git flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/App.js (hunk 1 above)
Comment (adamteddychang) [reply src/App.js]: Hello, thank you for your review. 
On the requirement page, it says that the default view should have the list of books and the form for adding the books. 
image
Can you clarify? Thank you

Comment (adamteddychang) [reply src/App.js]: I have updated the Routes as you suggested

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @adamteddychang,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**